docs/en/query_language/select.md

```diff
@@ -46,12 +46,16 @@ The FINAL modifier can be used only for a SELECT from a CollapsingMergeTree tabl
 
 ### SAMPLE Clause {#select-sample-clause}
 
-The `SAMPLE` clause allows for approximated query processing. Approximated query processing is only supported by the tables in the `MergeTree` family, and only if the sampling expression was specified during table creation (see [MergeTree engine](../operations/table_engines/mergetree.md)).
+The `SAMPLE` clause allows for approximated query processing (sampling). 
+
+Sampling is not performed on all the data, but only on a certain sample. It makes sense when you need to build reports on the fly on a large amount of data. For example, if you need to calculate statistics for all the visits, it is enough to execute the query on the 1/10 fraction of all the visits and then multiply the result by 10.
```

Comment:
* "Sampling ... sample" sounds weird.
* When reading second sentence about "large amount of data data" it feels like ClickHouse can't process huge datasets without sampling, while in practice it can. Actually it usually makes sense in several specific cases, for example:
  * When you have strict timing requirements (like <100ms) and at the same time you can't afford enough hardware resources to meet them.
  * When your raw data is not accurate in the first place and approximation doesn't make it much worse.
  * Business requirements need the result to be approximate, to make the service cost-effective or to sell exact results to premium users.